i18n/sv.toml

```
@@ -0,0 +1,85 @@
+[language]
+other = "sv"
+
+[links]
+  [links.get-involved]
+  other = "Bli involverad"
+
+  [links.get-in-touch]
+  other = "Kontakta oss"
+
+  [links.go-home]
+  other = "Hem"
+
+  [links.close-menu]
+  other = "Stäng menyn"
+
+  [links.documentation]
+  other = "Dokumentation"
+
+  [links.join-the-discussion]
+  other = "Gå med i diskussionen"
+
+[socials]
+  [socials.twitter]
+  other = "Twitter"
+
+  [socials.reddit]
+  other = "Reddit"
+
+  [socials.forums]
+  other = "Forum"
+
+  [socials.slack]
+  other = "Slack"
+
+  [socials.github]
+  other = "Github"
+
+[copyright]
+  [copyright.line1]
+  other = "© 2020 Rocky Linux-projektet. Alla rättigheter förbehållna."
+
+  [copyright.line2]
+  other = "CentOS är ett registrerat varumärke som tillhör Red Hat, Inc. Rocky Linux-projektet är inte anslutet till eller godkänt av Red Hat, Inc."
+
+[errors]
+  [errors.404]
+    [errors.404.title]
+    other = "Sidan hittades inte."
+
+    [errors.404.explanation]
+    other = "Den sida du försökte nå kunde inte hittas. Var god försök igen."
+
+[landing]
+  [landing.p1.heading]
+  other = "Rocky Linux"
+
+  [landing.p1.content]
+  other = "En gemenskapsdriven insats för att ge dig en företagsklass, produktionsklar Linux."
```

```suggestion
  other = "En gemenskapsdriven insats för att ge dig en produktionsklar Linux i företagsklass."
```

Once again, maybe even "En produktionsklar Linuxdistribution i företagsklass, utvecklad av frivilliga"?

---

thanks,